src/components/TextField/TextField.css.ts

```diff
@@ -0,0 +1,128 @@
+import { style } from '@vanilla-extract/css';
+import { recipe } from '@vanilla-extract/recipes';
+import { palette } from '@/styles/color';
+import { variants } from '../common/typography/Typography.css';
+
+export const inputTypeRecipe = recipe({
+  variants: {
+    status: {
+      default: 'default',
+      active: 'active',
+      error: 'error',
+      loading: 'loading',
+      success: 'success'
+    }
+  },
+  compoundVariants: [
+    {
+      variants: {},
+      style: {
+        vars: {
+          '--status-color': palette.gray300
```

Comment:
(마이너) vanillia extract에 createVars라는 함수를 사용하면 css 변수를 하드코딩하지 않아도 됩니다
``` typescript
const statusColor = createVars()
```
```suggestion
         [statusColor]: palette.gray300
```
```typescript
export const underbar = style({
  borderColor: statusColor
});
```

Reply:
헉 제가 너무 원하던 기능이었어요.. 알려주셔서 감사합니다!!